apps/src/netsim/NetSimNode.js

```diff
@@ -103,16 +89,7 @@ NetSimNode.prototype.getNodeType = function () {
  * @returns {string}
  */
 NetSimNode.prototype.getStatus = function () {
```

Comment:
From this context (now that it returns empty string), not sure what the method comment above `usually a string enum value` would mean in practice. Maybe could include an example or specific type?